src/onesignal.ts

```diff
@@ -87,3 +91,35 @@ export async function sendReadingReminderPush(
     'https://daily-now-res.cloudinary.com/image/upload/s--9vc188bS--/f_auto/v1712221649/1_smcxpz';
   await client.createNotification(push);
 }
+
+export async function sendReadingStreakReminderPush(
+  userIds: string[],
+  at: Date,
+): Promise<null | OneSignal.CreateNotificationSuccessResponse> {
+  if (!appId || !apiKey) return null;
+  const push = new OneSignal.Notification();
+  push.app_id = appId;
+  push.include_external_user_ids = userIds;
+  push.send_after = at.toISOString();
+  push.contents = {
+    en: readingStreakReminderContent,
+  };
+  push.headings = {
+    en: readingStreakReminderHeading,
+  };
+  push.url = addNotificationUtm(
+    process.env.COMMENTS_PREFIX,
+    'push',
+    // @TODO: check what this should be
+    'streak_reminder',
+  );
+
+  // @TODO: check what these should be
+  push.chrome_web_badge =
+    'https://daily-now-res.cloudinary.com/image/upload/v1672745846/public/dailydev.png';
+  // @TODO: check what these should be
+  push.chrome_web_icon =
+    'https://daily-now-res.cloudinary.com/image/upload/s--9vc188bS--/f_auto/v1712221649/1_smcxpz';
```

Comment:
@rebelchris could you check what these TODOs should be?

Reply:
@idoshamun I think you have more context on these, who do I check this with?

Reply:
we should ask design what image we should use for this. you can send them this link as reference but probably they'll come up with something related to streaks